src/api/app/controllers/webui/users/announcements_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Webui::Users::AnnouncementsController < Webui::WebuiController
```

Comment:
I think this should use `AnnouncementsController.update` in a `respond_to` block for `xhr`. Or if you want to in it's own message.

Reply:
Same here.

@hennevogel I've tried this but announcement routes are declared inside APIMatcher and it seems it accepts only xml and json format. If I'm not wrong, I should declare something like def self.public_or_about_path?(request) for exceptions.

So I wonder if it is really a good idea... thinking

Reply:
same as above...